.github/workflows/discover-new-releases.yaml

```diff
@@ -65,6 +65,63 @@ jobs:
           name: releases-${{ matrix.distro.name }}
           path: .github/workflows/scripts/list-releases/releases.json
 
+  check-splunk-releases:
+    name: Splunk OpenTelemetry Collector
+    strategy:
+      fail-fast: false
+      matrix:
+        distro:
+        - name: splunk-otel-collector
+          artifact_prefix: splunk-otel-collector
+          artifact_suffix: x86_64.rpm
+    runs-on: ubuntu-latest
+    outputs:
+      releases: ${{ steps.list-releases.outputs.releases }}
+    steps:
+      - name: Checkout repository
+        uses: actions/checkout@v4
+        with:
+          fetch-depth: 0
+      - name: Install dependencies
+        env:
+          NODE_MAJOR: 20
+        run: |
+          sudo apt-get update
+          sudo apt-get install -y ca-certificates curl gnupg
+          sudo mkdir -p /etc/apt/keyrings
+          curl -fsSL https://deb.nodesource.com/gpgkey/nodesource-repo.gpg.key | sudo gpg --dearmor -o /etc/apt/keyrings/nodesource.gpg
+          echo "deb [signed-by=/etc/apt/keyrings/nodesource.gpg] https://deb.nodesource.com/node_${NODE_MAJOR}.x nodistro main" | sudo tee /etc/apt/sources.list.d/nodesource.list
+          sudo apt-get update
+          sudo apt-get install nodejs -y
+      - name: Build list generator
+        working-directory: .github/workflows/scripts/list-releases
+        run: |
+          npm install
+          npm run build
+      - name: Look up ignored releases range
+        id: lookup-ignored-releases
+        shell: bash
+        working-directory: packages/otelbin-validation/src/assets
+        run: |
+          echo "range=$(jq -r '.["${{matrix.distro.name}}"].ignoredReleases' < supported-distributions.json)" | tee -a ${GITHUB_OUTPUT}
```

Comment:
**Issue:** `matrix.distro.name` resolves to `splunk-otel-collector` and that name does not match the name used in `supported-distributions.json` (`splunk`).

I'd suggest to keep it as `splunk`, i.e., to update the matrix name. Check out the CI step for `adot`, as that seems to be a more applicable copy/paste source.

Reply:
Changed `supported-distributions.json` to match.

Are there any more docs for this process, please?

Thank you :-)